stable/oauth2-proxy/templates/deployment.yaml

```diff
@@ -62,6 +69,7 @@ spec:
           - --google-service-account-json=/google/service-account.json
         {{- end }}
         {{- end }}
+        {{- if not .Values.isOpenshift }}
```

Comment:
if these values are not needed for openshift, we could check each one is set and only add them here if they are.